### ISSUE TITLE
[Suggestion] Connected textures for cincinnasite frames/glass

### ISSUE THREAD

**Issue by hiimagamer** [quiqueck/BetterNether#176]
### Description

**TLDR: I made a connected glass texturepack for cincinnasite frames/glass using optifine (don't know if it works the same with sodium ect)**

the [texturepack link](https://drive.google.com/file/d/1Z--oczHGCeZwrR0GQedrXSYpZvf-yE1W/view?usp=sharing)
I may delete this at some point if I need google drive space so feel free to archive.org this and ect _(and and cetera lol)_

tutorials I used incase you want to make your own or the developers want to create more:
[I think I used this one for the basics](https://www.youtube.com/watch?v=DrSZ5sNMUJM)
[and I think I used this one for how to get it working for later versions or a template or how to edit it or something](https://www.youtube.com/watch?v=-oQae-UNxHk)

epic backstory:
a little while ago I was play rlcraft and thought about having connected textures for the cincinnasite frames because they looked so cool (even with just the border)
so I looked into optifine connected textures and after a couple hours of debugging finally made the connected textures, I'm not familiar at all with minecraft code so I'm not sure if you can add this to the pack or not but may as well post it here
